website/src/css/pages/__case_studies.scss

```diff
@@ -72,6 +72,14 @@
                 font-size: 1.2rem;
                 line-height: 1.5;
             }
+
+            &--section {
```

Comment:
Ended up adding this to deal with a nested `h5` on `p` warning and figured that this was the least destructive way to not interfere with the existing case studies styles (border currently gets applied to all `p` elements under `case-studies__quote--in-content`)